confd/pkg/resource/template/template_funcs_test.go

```diff
@@ -58,7 +58,7 @@ func Test_BGPFilterBIRDFuncs(t *testing.T) {
 	testFilter.Spec = v3.BGPFilterSpec{
 		ImportV4: []v3.BGPFilterRuleV4{
 			{Action: "Accept", Source: "RemotePeers", Interface: "vxlan.calico", MatchOperator: "NotIn", CIDR: "55.4.0.0/16"},
-			{Action: "Reject", Source: "RemotePeers", MatchOperator: "NotIn", CIDR: "55.4.0.0/16"},
+			{Action: "Reject", Source: "RemotePeers", MatchOperator: "NotIn", CIDR: "55.4.0.0/16", PrefixLength: v3.BGPFilterPrefixLength{Min: 16, Max: 24}},
```

Comment:
Instead of replacing these test cases, can we copy/paste and add a second test case that includes the PrefixLength? That way we maintain coverage for both with and without the PrefixLength field set.

Reply:
I think we should also consider a test case for specifying a PrefixLength without a CIDR.

From what I can tell, it looks like the API will accept that input configuration but the code won't do anything with it. 

If this is a valid user input, then we should add a test here and make sure it puts out the right output. Otherwise, we should look at API validation to ensure that if a PrefixLength is given, so is a CIDR.